articles/aks/operator-best-practices-storage.md

```diff
@@ -38,7 +38,7 @@ The following table outlines the available storage types and their capabilities:
 
 The two primary types of storage provided for volumes in AKS are backed by Azure Disks or Azure Files. To improve security, both types of storage use Azure Storage Service Encryption (SSE) by default that encrypts data at rest. Disks cannot currently be encrypted using Azure Disk Encryption at the AKS node level.
 
-Azure Files are currently available in the Standard performance tier. Azure Disks are available in Standard and Premium performance tiers:
+Both Azure Files and Azure Disks are available in Standard and Premium performance tiers:
```

Comment:
Azure Files is available in a premium (SSD) tier: https://docs.microsoft.com/en-us/azure/storage/files/storage-how-to-create-premium-fileshare

However, I am not sure if this tier is an option for AKS. Please reject this PR if it is not an option for AKS.

Reply:
Thanks @andresnava, AKS supports premium tier Azure Files as of Kubernetes 1.13